src/couch/src/couch_db_updater.erl

```diff
@@ -1179,12 +1226,13 @@ copy_compact(Db, NewDb0, Retry) ->
 
     TaskProps0 = [
         {type, database_compaction},
+        {retry, (Retry /= nil)},
         {database, Db#db.name},
         {progress, 0},
         {changes_done, 0},
         {total_changes, TotalChanges}
     ],
-    case (Retry =/= nil) and couch_task_status:is_task_added() of
+    case (Retry /= nil) and couch_task_status:is_task_added() of
```

Comment:
Does `=/=` vs `/=` matter here? Or it just a matter of making it consistent with other `/= nil` checks?

Reply:
Nah. I had done some further refactoring trying a different set of optimizations that didn't pan out and this is just my mistake not reverting it correctly.